src/screens/Home/surveyList.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef } from 'react';
+
+import Slider from 'react-slick';
+
+import testSurvey from 'images/auth_background.png';
+import slickSettings from 'screens/Home/slickSettings';
+
+const SurveyList = () => {
+  const sliderRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('wheel', (e) => {
+      slideList(e.wheelDelta);
+    })
+  }, []);
+
+  const slideList = (wheel) => {
+    wheel > 0 ? (
+      sliderRef.current.slickNext()
+    ) : (
+      sliderRef.current.slickPrev()
+    )
+  }
+
+  return (
+    <div className="survey-list">
+      <Slider { ...slickSettings } ref={ sliderRef }>
+        <div>
```

Comment:
@rafayet-monon It seems fixed content 🤔
Should we make it more flexible content?